tools/test-backend

```diff
@@ -429,10 +423,12 @@ def main() -> None:
             reverse=options.reverse,
             keepdb=True,
         )
-        failures, failed_tests = test_runner.run_tests(
-            suites, full_suite=full_suite, include_webhooks=include_webhooks
+        failures = test_runner.run_tests(
+            suites,
+            failed_tests_path=FAILED_TEST_PATH,
```

Comment:
Could move the definition of `FAILED_TEST_PATH` to `zerver.lib.test_runner`, to avoid needing to pass it as an `Optional`.

Reply:
This will make `test-backend` depend on `zerver.lib.test_runner` in runtime to `get_failed_tests` with that path.

Reply:
Ah I see, we have an import mess that prevents that from working.